src/software/parameter/config/network.yaml

```diff
@@ -7,7 +7,8 @@ NetworkConfig:
       description: >-
           The multicast channel to join to communicate with the robots
     NetworkInterface:
-      default: "eth0"
+#      default: "enp4s0f1"
```

Comment:
Commented out code. If you want to have a list of example "default interface names", make it clear, but I assume it's just something that was overlooked.

Reply:
#1510 this will hopefully fix this, its a non-trivial problem, still trying to figure out a way to remove the dependency on this annoying interface string

Reply:
oops. fixed